skbio/stats/ordination/_principal_coordinate_analysis.py

```diff
@@ -24,7 +25,58 @@
 #   so, so I'm not doing that.
 
 
-class PCoA(Ordination):
+def _eigh(F_matrix, **ignored):
+    """Wrapper around np.linalg.eigh
+
+    The eigh method is described in [1].
+
+    Parameters
+    ----------
+    F_matrix : np.ndarray
+        The result of PCoABase._F_matrix
+
+    Returns
+    -------
+    array
+        The resulting eigenvalues.
+    array
+        The resulting eigenvectors.
+
+    References
+    ----------
+    .. [1] http://docs.scipy.org/doc/numpy/reference/generated/numpy.linalg.eigh.html  # noqa
+    """
+    return np.linalg.eigh(F_matrix)
+
+
+def _eigsh(F_matrix, k=10, **kwargs):
+    """Wrapper around scipy.sparse.linalg.eigsh
+
+    The eigsh method is described in [1].
+
+    Parameters
+    ----------
+    F_matrix : np.ndarray
+        The result of PCoABase._F_matrix
+    k : unsigned int
+        The number of eigenvectors and values to find.
+
+    Returns
+    -------
+    array
+        The resulting k eigenvalues.
+    array
+        The resulting k eigenvectors.
+
+    References
+    ----------
+    .. [1] http://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.linalg.eigsh.html  # noqa
+    """
+    val, vec = sp.sparse.linalg.eigsh(F_matrix, k=k, **kwargs)
+    return val, vec.T
```

Comment:
Is this transpose correct? It makes `eigsh` inconsistent with `eigh` and so it'll break the rest of the code if used.